adapter/internal/oasparser/constants/constants.go

```diff
@@ -98,7 +99,20 @@ const (
 	HTTP                 string = "HTTP"
 	WS                   string = "WS"
 	WEBHOOK              string = "WEBHOOK"
+	SSE                  string = "SSE"
 	Prototyped           string = "prototyped"
 	TemplateEndpointType string = "TEMPLATE"
 	InlineEndpointType   string = "INLINE"
 )
+
+// Constants used for version identification of API definitions
+const (
+	Swagger      string = "swagger"
+	OpenAPI      string = "openapi"
+	AsyncAPI     string = "asyncapi"
+	Swagger2     string = "swagger_2"
```

Comment:
let's use openapi2 instead of swagger

Reply:
As per the [specification](https://swagger.io/docs/specification/2-0/basic-structure/) the file starts with `swagger: "2.0"` therefore I think it is clearer this way. Also the golang files related to swagger 2 are also starts with `swagger`.